opentelemetry-otlp/src/metric.rs

```diff
@@ -303,13 +386,83 @@ impl MetricsExporter {
             }
         });
 
-        Ok(MetricsExporter {
+        Ok(MetricsExporter::Tonic {
             sender: Mutex::new(sender),
             temporality_selector,
             aggregation_selector,
             metadata: tonic_config.metadata.take(),
         })
     }
+
+    /// Create a new OTLP HTTP metrics exporter.
+    #[cfg(feature = "http-proto")]
+    pub fn new_http(
+        export_builder: HttpExporterBuilder,
+        temporality_selector: Box<dyn TemporalitySelector>,
+        aggregation_selector: Box<dyn AggregationSelector>,
+    ) -> Result<MetricsExporter> {
+        let config = export_builder.exporter_config;
+        let http_config = export_builder.http_config;
+        let _endpoint = match std::env::var(OTEL_EXPORTER_OTLP_METRICS_ENDPOINT) {
+            Ok(val) => val,
+            Err(_) => format!("{}{}", config.endpoint, "/v1/metrics"),
+        };
+
+        let url: Uri = config
+            .endpoint
+            .parse()
+            .map_err::<crate::Error, _>(Into::into)?;
+
+        let _timeout = match std::env::var(OTEL_EXPORTER_OTLP_METRICS_TIMEOUT) {
```

Comment:
not using `_timeout`?

Reply:
Yes as of now, it is using the timeout from config file (consistent with the otlp-http span exporter). Thought of creating separate PR to read _timeout from environment variable.